lib/game.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  def initialize(player_one, player_two, board)
+    @player_one = player_one
+    @player_two = player_two
+    @board = board
+  end
+
+  def switch_player(active_player)
+    active_player == @player_one ? @player_two : @player_one
```

Comment:
It's not necessary to have a parameter here
Update it like this
```ruby
def switch_player
   @active_player = @active_player == @player_one ? @player_two : @player_one
end
```
And you initialize (into the constructor) `@active_player` randomly with either `@player_one` or `@player_two`.